eureka/S5_lightcurve_fitting/s5_fit.py

```diff
@@ -0,0 +1,76 @@
+import numpy as np
+from ..lib import manageevent as me
+from ..lib import readECF as rd
+import glob, os
+from ..lib import sort_nicely as sn
+import matplotlib.pyplot as plt
+from importlib import reload
+
+def fitJWST(eventlabel, workdir, speclc_dir, fit_par, run_par_file, meta):
+    # load savefile
+    if meta == None:
+        meta = me.load(speclc_dir + '/S4_' + eventlabel + '_Meta_Save.dat')
+
+    #read in run params
+    import eureka.S5_lightcurve_fitting.parameters as p
+    reload(p)
+    run_par=p.Parameters(param_file=run_par_file)
+    # Create directories for Stage 4 processing
+    files = glob.glob(os.path.join(speclc_dir + "/speclc", "*.txt"))
+    files = sn.sort_nicely(files)
+    if run_par.run_verbose.value:
+        print(files)
+    t0_offset = run_par.toffset.value#59694
+    for f in [files[0]]:
```

Comment:
Why is this written as a for-loop if you're just taking the first element of files? I guess the interpretation of what you're doing here depends on what the files you're reading in are

Reply:
Yeah, again, this is some code that was only there when Stage 5 was running on files not output from the rest of Eureka.